src/many-client/src/client/blocking.rs

```diff
@@ -0,0 +1,62 @@
+use std::sync::Arc;
+
+use many_identity::{Address, Identity};
+use many_modules::base::Status;
+use many_protocol::{RequestMessage, ResponseMessage};
+use many_server::ManyError;
+use minicbor::Encode;
+use reqwest::IntoUrl;
+use tokio::runtime::Runtime;
+
+use crate::ManyClient as AsyncClient;
+
+#[derive(Debug, Clone)]
+pub struct ManyClient<I: Identity> {
+    client: AsyncClient<I>,
+    runtime: Arc<Runtime>,
+}
+
+impl<I: Identity> ManyClient<I> {
+    pub fn new<S: IntoUrl>(url: S, to: Address, identity: I) -> Result<Self, String> {
+        let client = AsyncClient::new(url, to, identity)?;
+        let runtime = Arc::new(
+            tokio::runtime::Builder::new_current_thread()
```

Comment:
As mentioned on Slack, we need to be careful since I don't think creating a new runtime from a runtime thread is allowed.

We need to ensure the proper context, sync or async, is used in `many-framework`, i.e., use the sync client in a sync application and the async client in an async application.

It would be nice to be able to enforce this on the user.